src/containers/App/index.js

```diff
@@ -26,8 +26,8 @@ const App = () => (
     <MainWrapper>
       <Switch>
         <Route exact path="/" component={Home} />
-        <Route exact path="/auth" component={Auth} />
-        <Route exact path="/daily_reports" component={DailyReports} />
+        <Route path="/auth" component={Auth} />
+        <Route path="/Member" location={location} component={Member} />
```

Comment:
The route path should be lowercase. 
`<Route path="/member" location={location} component={Member} />`